LICENSE

```diff
@@ -2531,3 +2531,101 @@ This product bundles 'zlib' within 'OpenCV' under the following license:
   The data format used by the zlib library is described by RFCs (Request for
   Comments) 1950 to 1952 in the files http://tools.ietf.org/html/rfc1950
   (zlib format), rfc1951 (deflate format) and rfc1952 (gzip format).
+
+This product bundles 'TinyXml2' within 'AWS SDK for C++' under a zlib license:
+
+Original code by Lee Thomason (www.grinninglizard.com)
+
+This software is provided 'as-is', without any express or implied
+warranty. In no event will the authors be held liable for any
+damages arising from the use of this software.
+
+Permission is granted to anyone to use this software for any
+purpose, including commercial applications, and to alter it and
+redistribute it freely, subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must
+not claim that you wrote the original software. If you use this
+software in a product, an acknowledgment in the product documentation
+would be appreciated but is not required.
+
+2. Altered source versions must be plainly marked as such, and
+must not be misrepresented as being the original software.
+
+3. This notice may not be removed or altered from any source
+distribution.
+
+
+This product bundles 'cJSON' within 'AWS SDK for C++' under an MIT license:
```

Comment:
The `NOTICE.txt` of aws-sdk-cpp says that they bundle JsonCpp, but this is no longer true, as evidenced by the code and the `1.5.0` part of https://github.com/aws/aws-sdk-cpp/blob/master/CHANGELOG.md
They now bundle cJSON, so I have included the license for that here.